src/com/company/topic7/Main.java

```diff
@@ -0,0 +1,27 @@
+package com.company.topic7;
+
+import java.io.File;
+import java.io.IOException;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        String adresaFisierului = "D:\\Proiectele mele\\Test_automation_course\\src\\com\\company\\topic7\\fisierulDeLucru.txt";
```

Comment:
better to user relatve paths like src\\com\\company\\topic7\\fisierulDeLucru.txt